rtl/include/eclhelper.hpp

```diff
@@ -721,7 +721,8 @@ interface ICodeContext : public IResourceContext
 
     // Exception handling
 
-    virtual void addWuException(const char * text, unsigned code, unsigned severity, unsigned audience, const char * source) = 0;
+    virtual void addWuException(const char * text, unsigned code, unsigned severity, const char * source) = 0;
+    virtual void addWuExceptionEx(const char * text, unsigned code, unsigned severity, unsigned audience, const char * source) = 0;
```

Comment:
This new function needs to be added at the end of the interface (otherwise calls to following functions will no longer be valid)